src/common/Cargo.toml

```diff
@@ -21,6 +21,7 @@ arrow-buffer = "45"
 arrow-schema = "45"
 async-trait = "0.1"
 auto_enums = "0.8"
+bincode = { version = "=2.0.0-rc.3", features = ["serde"] }
```

Comment:
Required by `LocalStateStore::init`'s refactor.
It is required to import the `Decode and Encode` macros, for `EpochPair` in the `common` crate.

Reply:
Refactor is done externally. We can remove this dependency.